docs/rules/jsx-boolean-value.md

```diff
@@ -6,27 +6,29 @@
 
 ## Rule Details
 
-This rule takes one argument. If it is `"always"` then it warns whenever an attribute is missing its value. If `"never"` then it warns if an attribute has a `true` value. The default value of this option is `"never"`.
+This rule takes two arguments. If the first argument is `"always"` then it warns whenever an attribute is missing its value. If `"never"` then it warns if an attribute has a `true` value. The default value of this option is `"never"`.
 
-The following patterns are considered warnings when configured `"never"`:
+The second argument is optional: if provided, it must be an object with a `"never"` property (if the first argument is `"always"`), or an `"always"` property (if the first argument is `"never"`). This property’s value must be an array of strings representing prop names.
+
+The following patterns are considered warnings when configured `"never"`, or with `"always", { "never": ["personal"] }`:
```

Comment:
I'm on the fence about whether this should be `never`/`always`, or something more general like `exceptions` or `except`. As it is written here, it is possible to have both `never` and `always` which would be a little weird. I think it is fine, but I think a unified option might make the documentation a little clearer and the code a small amount simpler. What do you think?